numpy/lib/_arraysetops_impl.py

```diff
@@ -180,7 +180,8 @@ def unique(ar, return_index=False, return_inverse=False,
         .. versionadded:: 1.13.0
 
     equal_nan : bool, optional
-        If True, collapses multiple NaN values in the return array into one.
+        If True, collapses multiple NaN values in the return array into one
+        and it is put to the end of the sorted unique values.      
```

Comment:
I think the addition belongs as a separate sentence in the Notes, since sort order is not connected to the `equal_nan` option.
> Like `np.sort`, NaN will sort to the end of the values.

Extra points for adding `sort` to the See Also section.